fusor-ember-cli/app/templates/components/text-f.hbs

```diff
@@ -1,4 +1,4 @@
-{{#base-f label=label labelSize=labelSize inputSize=inputSize unitsSize=unitsSize unitsLabel=unitsLabel help-inline=help-inline errors=errors isRequired=isRequired hasError=hasError}}
+{{#base-f label=label labelSize=labelSize inputSize=inputSize unitsSize=unitsSize unitsLabel=unitsLabel help-inline=help-inline errors=errors isRequired=isRequired hasError=hasError helpText=helpText}}
 
```

Comment:
This is perfect example to demonstrate a component within a component. Since component  `text-f` uses component `base-f`, it needs to pass in `helpText=helpText` because components are **isolated**. They only see what's passed into them.